buildscripts/make_dependencies.sh

```diff
@@ -26,9 +28,18 @@ else
   pushd $DOWNLOAD_DIR/protobuf-${PROTOBUF_VERSION}
   ./autogen.sh
   # install here so we don't need sudo
-  ./configure --disable-shared --prefix="$INSTALL_DIR"
+  ./configure CFLAGS=-m$ARCH CXXFLAGS=-m$ARCH LDFLAGS=-m$ARCH --disable-shared \
```

Comment:
This LDFLAGS looks wrong. I don't think that's a valid linker flag.

Reply:
You're right that is not needed.